ci_framework/roles/dlrn_promote/tasks/get_hashes.yml

```diff
@@ -0,0 +1,21 @@
+---
+- name: Copy delorean.repo to dlrn_promote workspace
+  ansible.builtin.fetch:
+    src: "{{ cifmw_dlrn_promote_dlrnrepo_path }}"
+    dest: "{{ cifmw_dlrn_promote_workspace }}/"
+    flat: true
+
+- name: Get baseurls from delorean.repo
+  ansible.builtin.shell:
```

Comment:
Is there a cleaner way to do this using ansible native tools?

Reply:
Done